app/src/main/java/org/simple/clinic/signature/SignatureActivity.kt

```diff
@@ -49,9 +48,25 @@ class SignatureActivity : AppCompatActivity(), SignatureUiActions {
     )
   }
 
+  private lateinit var binding: ActivitySignatureBinding
+
+  private val signatureRoot
+    get() = binding.signatureRoot
+
+  private val acceptSignature
+    get() = binding.acceptSignature
+
+  private val drawSignatureFrame
+    get() = binding.drawSignatureFrame
+
+  private val clearSignature
+    get() = binding.clearSignature
+
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentView(R.layout.activity_signature)
+
+    binding = ActivitySignatureBinding.inflate(layoutInflater)
```

Comment:
Can we nullify binding when the screen is destroyed?

Reply:
We don't need to do that for Activities. We only need to do that for Fragments and Views.

Reply:
I see. Why is that activities don't need?

Reply:
You can refer to the official [docs](https://developer.android.com/topic/libraries/view-binding). But essentially Fragments can outlive their views, so to avoid any memory leaks we clear the binding.